fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -114,6 +119,12 @@ public class HiveMetaStoreCache {
     private final ExecutorService refreshExecutor;
     private final ExecutorService fileListingExecutor;
 
+    // cache from <dbname-tblname> -> <num of partitions>
+    private LoadingCache<PartitionNumCacheKey, Integer> partitionNumCache;
+
+    // cache from <dbname-tblname-filter> -> <values of partitions>
+    private LoadingCache<FilterPartitionValueCacheKey, Map<Long, PartitionItem>> filterPartitionValuesCache;
```

Comment:
- These two caches should can be disable by size = 0 or `enable` flag, and the default value may be disabled better. 
-  Can we merge `partitionValuesCache` to `filterPartitionValuesCache`?